### PR TITLE
Task 3 Selenium WE

### PR DESCRIPTION
**Pull request by OleksandrBerezhnyi**

### CODE REVIEW

File: src/main/java/ua/yakaboo/Pages/LoginPage.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package ua.yakaboo.Pages;
+
+public class LoginPage {
+//
+//    @FindBy ()
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/yakaboo/Pages/LoginPage.java]: Remove this comments please

File: src/test/java/ua/yakaboo/Tests/HomePageTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomePageTest extends BaseTests {
+
+    @Test
+    public void homePageIsDisplayed() {
+        String title = driver.getTitle();
+        Assert.assertTrue(title.equals("Yakaboo — Інтернет-магазин книг, подарунків і дитячих товарів. Купити книги та подарунки по найкращим в Україні цінами."));
```
Comment (DmitriyBondarenko) [inline src/test/java/ua/yakaboo/Tests/HomePageTest.java]: `Assert.assertEquals(title, "");`

File: src/main/java/ua/yakaboo/Pages/CartPage.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package ua.yakaboo.Pages;
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/yakaboo/Pages/CartPage.java]: All package name should be from lower case

File: src/main/java/ua/yakaboo/Pages/HeaderPage.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package ua.yakaboo.Pages;
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/yakaboo/Pages/HeaderPage.java:1]: All package name should be from lower case

File: src/main/java/ua/yakaboo/Pages/HomePage.java [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package ua.yakaboo.Pages;
```
Comment (DmitriyBondarenko) [inline src/main/java/ua/yakaboo/Pages/HomePage.java:1]: All package name should be from lower case

File: src/test/java/ua/yakaboo/Tests/BaseTests.java [hunk 6]
```diff
@@ -1,4 +1,4 @@
-package ua.yakaboo;
+package ua.yakaboo.Tests;
```
Comment (DmitriyBondarenko) [inline src/test/java/ua/yakaboo/Tests/BaseTests.java]: From lower case please

**Review by DmitriyBondarenko — CHANGES_REQUESTED** [review]
comments

File: src/main/java/ua/yakaboo/Pages/CartPage.java (hunk 3 above)
Comment (OleksandrBerezhnyi) [reply src/main/java/ua/yakaboo/Pages/CartPage.java]: Updated

File: src/test/java/ua/yakaboo/Tests/BaseTests.java (hunk 6 above)
Comment (OleksandrBerezhnyi) [reply src/test/java/ua/yakaboo/Tests/BaseTests.java]: Updated

File: src/test/java/ua/yakaboo/Tests/HomePageTest.java (hunk 2 above)
Comment (OleksandrBerezhnyi) [reply src/test/java/ua/yakaboo/Tests/HomePageTest.java]: Fixed

File: src/main/java/ua/yakaboo/Pages/LoginPage.java (hunk 1 above)
Comment (OleksandrBerezhnyi) [reply src/main/java/ua/yakaboo/Pages/LoginPage.java]: Removed

File: src/test/java/ua/yakaboo/tests/BaseTests.java [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import utils.driverUtils.DriverProvider;
+
+import static utils.PropertyReader.getProperty;
+
+public class BaseTests {
+
+    public WebDriver driver;
```
Comment (DmitriyBondarenko) [inline src/test/java/ua/yakaboo/tests/BaseTests.java:12]: `protected DriverProvider driver;`

File: src/main/java/utils/driverUtils/DriverProvider.java [hunk 8]
```diff
@@ -0,0 +1,46 @@
+package utils.driverUtils;
```
Comment (DmitriyBondarenko) [inline src/main/java/utils/driverUtils/DriverProvider.java]: just `driver`

Comment (OleksandrBerezhnyi) [reply src/main/java/utils/driverUtils/DriverProvider.java]: Renamed

**Pull request merged**